.github/workflows/main.yml

```diff
@@ -14,9 +14,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        otp: [22.3.4.14, 23.0.4, 23.1.5.0, 23.2.1.0, 23.2.7.0]
+        otp: [22.3.4.14, 23.0.4, 23.1.5.0, 23.2.1.0, 23.2.7.0, 23.3.4.0, 'quay-23.3.4']
     container:
-      image: erlang:${{ matrix.otp }}-alpine
+      image: ${{ matrix.otp != 'quay-23.3.4' && format('{0}:{1}-{2}', 'erlang', matrix.otp, 'alpine') || 'quay.io/travelping/alpine-erlang:23.3.4' }}
```

Comment:
too complex, just switch all matrix steps to our images and remove the versions where we have no images for.

Reply:
Done